datashader/__init__.py

```diff
@@ -1,5 +1,3 @@
-from __future__ import absolute_import
```

Comment:
Perhaps instead of removing `__future__` imports altogether, it would be better to replace them with:
```py
from __future__ import annotations
```
for the sake of cohesion between holoviz projects.

Reply:
I was going to say that this is out of scope of the current changes as I am looking back fixing historic problems rather than looking forward making improvements, but I may as well do this now as it affects exactly the same lines of code. So yes, I will.